deps/hdr_histogram/hdr_histogram.c

```diff
@@ -392,16 +393,16 @@ int hdr_init(
         return r;
     }
 
-    counts = (int64_t*) calloc((size_t) cfg.counts_len, sizeof(int64_t));
+    counts = (int64_t*) hdr_calloc((size_t) cfg.counts_len, sizeof(int64_t));
```

Comment:
maybe it would be safer to use macros?
i.e. `#define malloc hdr_malloc`

what i'm thinking of is the chance we'll merge an upgrade from upstream, and miss one use of malloc of free, so it'll pass to a different allocator (which in most cases would be the same allocator, just without the zmalloc counting).
this could be VERY dangerous since there's a big chance we won't notice it.